lib/ts/recipe/passwordless/types.ts

```diff
@@ -241,7 +215,7 @@ export type TypeNormalisedInput = (
     // By default (=undefined) it is done in the Core
     getCustomUserInputCode?: (userContext: any) => Promise<string> | string;
 
-    emailDelivery: EmailDeliveryTypeInput<TypeEmailDeliveryTypeInput>;
+    emailDelivery: EmailDeliveryTypeInput<TypePasswordlessEmailDeliveryTypeInput>;
```

Comment:
We should move this into where the `CONTACT_METHOD` is EMAIL or EMAIL_OR_PHONE. Make this optional in there and make `createAndSendCustomEmail` optional too. If the user has not provided either, you should call our API (this is a new thing) -> you will have to make an API for this, similar to how we have for password reset and email verification.

Reply:
> If the user has not provided either, you should call our API (this is a new thing) -> you will have to make an API for this, similar to how we have for password reset and email verification.

This part is remaining. Just FYI when you are reviewing